### ISSUE TITLE
Fails in generateReports

### ISSUE THREAD

**Issue by platise** [taskjuggler/TaskJuggler#272]
Hi,

with the very latest 3.7.1 #fbb669bdbce50a5f3 got this error, which projects works well under 3.6.0:
~~~~~
	from /var/lib/gems/2.7.0/gems/taskjuggler-3.7.1/lib/taskjuggler/Project.rb:760:in `generateReports'
	from /var/lib/gems/2.7.0/gems/taskjuggler-3.7.1/lib/taskjuggler/TaskJuggler.rb:158:in `generateReports'
	from /var/lib/gems/2.7.0/gems/taskjuggler-3.7.1/lib/taskjuggler/apps/Tj3.rb:216:in `appMain'
	from /var/lib/gems/2.7.0/gems/taskjuggler-3.7.1/lib/taskjuggler/Tj3AppBase.rb:160:in `main'
	from /var/lib/gems/2.7.0/gems/taskjuggler-3.7.1/lib/tj3.rb:16:in `<top (required)>'
	from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
	from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:72:in `require'
	from /var/lib/gems/2.7.0/gems/taskjuggler-3.7.1/bin/tj3:4:in `<top (required)>'
	from /usr/local/bin/tj3:23:in `load'
	from /usr/local/bin/tj3:23:in `<main>'
~~~~~
On a new Ubuntu 20lts system I've used:
~~~
gem install rake mail rspec term-ansicolor simplecov
~~~
and built as `gem build taskjuggler.gemspec` and then installed.

Is it my installation broken or something is missing in 3.7.1 ?

BR Uros

**Comment by scrapper** [comment]
Please include the full error message. This is just the stack trace without the actual error message.

**Comment by platise** [comment]
There was no other error on stderr, this was all.

Meanwhile I tried online version `gem install taskjuggler` which is also 3.7.1 and it works, so obviously I am compiling it wrong, or have something missing. With this pre-compiled latest version I have noticed that output directory for
~~~
project IoT "IoT" "1.0" 2012-11-15 - 2022-01-01 {
    include "project.tji"
    outputdir "Reports/IoT"
}
~~~
does not work for this specific report:
~~~
export "Internal/project" {
    formats tjp
    definitions tasks
    taskattributes complete, flags, note, booking
    resourceattributes -
}
~~~
is generated under `Internal/project.tji` instead in the `Reports/IoT/Internal/projects.tji` while all other reports are successfully generated under `Reports/IoT/...` as requested.
